Gemfile

```diff
@@ -36,15 +38,18 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+# se usa intslata la ge,a de rubocop emn el proyecto
+gem 'rubocop', require: false
```

Comment:
Add the rubocop version. For example:
`gem 'rubocop', '~> 0.59.0', require: false`